src/pages/pass-finalisation/pass-finalisation.ts

```diff
@@ -23,8 +55,63 @@ export class PassFinalisationPage extends BasePageComponent {
     super(platform, navCtrl, authenticationProvider);
   }
 
+  ngOnInit(): void {
+    this.pageState = {
+      candidateName$: this.store$.pipe(
+        select(getCurrentTest),
```

Comment:
There is a `getTests` selector introduced which you can find in the `testsReducer`. Using that before getting the current test is necessary.